### PR TITLE
play 3 upgrade

### PR DESCRIPTION
**Pull request by dblatcher**
## What does this change?

upgrades to play 3
 - upgrade java from corretto v8  -> v11
 - upgrade scala from  2.12.16 -> 2.13.0
 - upgrade sbt 1.7.1 -> 1.9.8
 - akka -> pekko source code and config (max-header-size) changes
 - source code changes for scala upgrade
 - sbt-web plugin removed - there isn't a version compatible with play-3, but the sbt-digest and sbt-gzip plugins (built to work with sbt-web) still work - it seems the functionlity of sbt-web has been incorporated into play 3

## How to test
Have [deployed](https://riffraff.gutools.co.uk/deployment/view/05dad197-e9a6-45dc-8666-8cebc7d83adf) to CODE, [test article](https://viewer.code.dev-gutools.co.uk/preview/uk-news/2015/aug/11/london-underground-strike-august-night-tube-dispute) would still render and client-side code still worked:

## How can we measure success?

reduction in vulnerabilities.

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -7,8 +7,8 @@ play {
   }
 
   server {
-    akka {
-      max-header-value-length = 16k
+    pekko {
```
Comment (davidfurey) [inline conf/application.conf:10]: Good spot!

File: conf/application.conf [hunk 2]
```diff
@@ -17,4 +17,6 @@ play {
   i18n.langs = [ "en" ]
 }
 
+# max-header-size
```
Comment (davidfurey) [inline conf/application.conf]: I think this might be a note that got left behind?

File: build.sbt [hunk 3]
```diff
@@ -24,34 +23,32 @@ lazy val root = (project in file("."))
      )
   )
 
-scalaVersion := "2.12.16"
+scalaVersion := "2.13.0"
 
 val awsVersion = "1.12.129"
 
+// Until all dependencies are on scala-java8-compat v1.x, this avoids unnecessary fatal eviction errors
+ThisBuild / libraryDependencySchemes += "org.scala-lang.modules" %% "scala-java8-compat" % VersionScheme.Always
```
Comment (davidfurey) [inline build.sbt]: Does this need to be here and in plugins.sbt?

File: conf/application.conf (hunk 2 above)
Comment (dblatcher) [reply conf/application.conf]: yes, thanks - have removed

File: build.sbt (hunk 3 above)
Comment (dblatcher) [reply build.sbt]: Looks like we don't need this or `libraryDependencySchemes` line in plugins.sbt - it was needed to compile after upgrading play to  2.9, but works locally without them on play 3.0 - will check on CODE again once built

**Review by davidfurey — APPROVED** [review]
lgtm

**Pull request merged**